Snake/BusinessLayer/GameSnake.cs

```diff
@@ -7,27 +7,30 @@
 namespace BusinessLayer
 {
     public class GameSnake
-    {
+    {
         enum Direction {right, down, left, up};
         Direction directionFacing;
         private int snakeLength;
         private List<Point> currentPosition;
```

Comment:
if the list and length is only required in the context of the method there is no point in it being a private member on the class, putting here implies can be shared within and instance across different methods, is that the case?